### ISSUE TITLE
Doesn't allow a root "output_dir"

### ISSUE THREAD

**Issue by stormlifter** [mintchaos/django_compressor#21]
I'd like it to output directly to the media_url, but something causes an issue when I try this. Haven't had time to look through and find out why.

**Comment by jezdez** [comment]
Is there something like traceback?

**Comment by stormlifter** [comment]
Caught Exception while rendering: Traceback (most recent call last):
  File "C:\Python26\lib\site-packages\compressor\templatetags\compress.py", line 38, in render
    output = compressor.output()
  File "C:\Python26\lib\site-packages\compressor__init__.py", line 181, in output
    ret.append(subnode.output())
  File "C:\Python26\lib\site-packages\compressor__init__.py", line 175, in output
    return super(CssCompressor, self).output()
  File "C:\Python26\lib\site-packages\compressor__init__.py", line 135, in output
    self.save_file()
  File "C:\Python26\lib\site-packages\compressor__init__.py", line 127, in save_file
    if self.storage.exists(self.new_filepath):
  File "C:\Python26\lib\site-packages\django\core\files\storage.py", line 198, in exists
    return os.path.exists(self.path(name))
  File "C:\Python26\lib\site-packages\django\core\files\storage.py", line 214, in path
    raise SuspiciousOperation("Attempted access to '%s' denied." % name)
SuspiciousOperation: Attempted access to '/css/41b9f5ca7743.css' denied.

**Comment by stormlifter** [comment]
This would be really nice to have fixed. having a root output directory makes it really convenient since I don't have to move my images or anything to another directory.

**Comment by jezdez** [comment]
This sounds a lot like you confuse MEDIA_ROOT and MEDIA_URL, where the first is the actual file system location to save the files to and the latter the URL path to that directory. In other words, SuspiciousOperation is raised when you don't have permission to write to a directory, in your case to the filesystem directory "/css".

**Comment by stormlifter** [comment]
I'll test this once I get a chance. Thanks for the feedback. I guess I'll just add a setting to control if I want it to be at the MEDIA_ROOT or not.

**Comment by stormlifter** [comment]
Actually I looked through and it isn't working right.

OUTPUT_DIR = getattr(settings, 'COMPRESS_OUTPUT_DIR', '')

That line has to have something specified, if I put blank it errors out. What would I put in there to make it save to the base MEDIA_ROOT. I tried putting MEDIA_ROOT, but that puts down the full windows directory and it doesn't pull up the files correctly.

Ideally putting nothing in there would just have it store to the relative location to the MEDIA_ROOT just like putting the 'CACHE' causes it to put a folder in the root and put things there.
